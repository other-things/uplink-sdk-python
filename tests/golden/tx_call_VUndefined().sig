r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 57241446333497058700828504906977167385460484155620650047038271832801646862861
