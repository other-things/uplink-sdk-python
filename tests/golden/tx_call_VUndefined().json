{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "tag": "VUndefined"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE01NzI0MTQ0NjMzMzQ5NzA1ODcwMDgyODUwNDkwNjk3NzE2NzM4NTQ2MDQ4NDE1NTYyMDY1MDA0NzAzODI3MTgzMjgwMTY0Njg2Mjg2MQ=="
}